cbf/v1/cbf-v1.0.0.yaml

```diff
@@ -0,0 +1,556 @@
+          maxLength: 50
+        transportCallReference:
+          type: string
+          description: |-
+            A carrier definied reference to a TransportCall.
+
+            In the case the Means of Transport is a Vessel and the facility is a Port/Terminal - this reference should be considered a Terminal Call Reference
+          example: da97d1e3-e546-4647-891b-3f57e04b886b
+        carrierImportVoyageNumber:
+          type: string
+          example: 2103N
+          description: The identifier of an import voyage. The carrier-specific identifier of the import Voyage.
+          maxLength: 50
+        carrierExportVoyageNumber:
+          type: string
+          description: The identifier of an export voyage. The carrier-specific identifier of the export Voyage.
+          example: 2103S
+          maxLength: 50
+          pattern: ^\S+(\s+\S+)*$
+        universalImportVoyageReference:
+          type: string
+          description: 'A global unique voyage reference for the import Voyage, as per DCSA standard, agreed by VSA partners for the voyage.'
+          pattern: '^\d{2}[0-9A-Z]{2}[NEWS]$'
+          example: 2103N
+        universalExportVoyageReference:
+          type: string
+          description: 'A global unique voyage reference for the export Voyage, as per DCSA standard, agreed by VSA partners for the voyage.'
+          pattern: '^\d{2}[0-9A-Z]{2}[NEWS]$'
+          example: 2103N
+        location:
+          oneOf:
+            - $ref: ../../models/UnLocationLocation.yaml
+            - $ref: ../../models/FacilitySMDGLocation.yaml
+        loadLadenContainers:
+          description: Lists all the `LADEN` containers being loaded.
+          type: array
+          uniqueItems: true
+          minItems: 0
+          items:
+            $ref: '#/components/schemas/LadenContainer'
+            x-stoplight:
+              id: xwd9wgupwk8m7
+        loadEmptyContainers:
+          description: Lists all the `EMPTY` containers being loaded
+          type: array
+          minItems: 0
+          uniqueItems: true
+          items:
+            $ref: '#/components/schemas/EmptyContainer'
+            x-stoplight:
+              id: chwpisamzgh4b
+        loadBreakBulk:
+          type: array
+          minItems: 0
+          uniqueItems: true
+          items:
+            $ref: '#/components/schemas/NonContainer'
+            x-stoplight:
+              id: frcnb8jwobq7a
+        loadOutOfGauge:
+          type: array
+          minItems: 0
+          uniqueItems: true
+          items:
+            $ref: '#/components/schemas/NonContainer'
+            x-stoplight:
+              id: 02vefvs2x9qbz
+      required:
+        - location
+    LadenContainer:
+      title: LadenContainer
+      x-stoplight:
+        id: kdfqf0ojv0wqd
+      type: object
+      description: A CBF specification for a particular container type.
+      properties:
+        equipmentSizeCode:
+          type: string
+          example: '20'
```

Comment:
Further down you have this as an enum

Reply:
Type `string` with values limited via `enum` , not sure what you mean?

Reply:
Further down in `DischargeLadenContainer` and in `EmptyContainer` you have defined this field as an enum. I assume it should be defined in the same way everywhere..?

Reply:
Yes, this is a type that indicates it's a `string` field. However, if this is an incorrect interpretation or a misuse, do let me know.